### ISSUE TITLE
Disable bar chart darken on hover

### ISSUE THREAD

**Issue by chinaowl** [eventbrite/britecharts#485]
Question: is it possible to disable the bar chart darkening on hover? I actually want the opposite effect, where the bar that's being hovered over stays the same but the other bars get darker, which I can do with a custom mouseover event. But it seems like the darkening will always happen because it comes after (in the source code):

```
function handleMouseOver(e, d, chartWidth, chartHeight) {
    dispatcher.call('customMouseOver', e, d, d3Selection.mouse(e), [chartWidth, chartHeight]);
    d3Selection.select(e).attr('fill', ({name}) => d3Color.color(colorMap(name)).darker());
}
```

EDIT: I've come up with a workaround that _mostly_ works. On the first hover, all bars darken (haven't figured out why), but on subsequent hovers, it has the effect that I want. https://codepen.io/chinaowl/pen/jYejrd

**Comment by Golodhros** [comment]
Hi @chinaowl, thanks for opening an issue!
I do agree with you, this shouldn't be a fixed behavior but an optional one.

I would suggest to basically remove that line that darkens the bar color and the one that restores it's color.

Thanks!

**Comment by dalerasrorov-eb** [comment]
@chinaowl, I implemented something that might be handy https://github.com/eventbrite/britecharts/pull/500

**Comment by chinaowl** [comment]
@dalerasrorov-eb awesome, thank you!

**Comment by dalerasrorov-eb** [comment]
@chinaowl this weekend or next Monday!

**Comment by dalerasrorov-eb** [comment]
Closing this @chinaowl, the functionality was released in `2.5.0` and the new functionality to apply a custom behavior to highlighted bars is ready https://github.com/eventbrite/britecharts/pull/503 and will be released later. Thanks for opening the issue!
